bondcpp/package.xml

```diff
@@ -1,4 +1,6 @@
-<package>
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
```

Comment:
I had error on `debian:stretch`, `ubuntu:yakkety`and `ubuntu:zesty`, and this causes error on finding `nodelet` by `pluginlib`.

- https://travis-ci.org/ros-perception/perception_pcl/jobs/263001942
- https://travis-ci.org/ros-perception/perception_pcl/jobs/263001943

```
root@536df0d8dd4b:~/catkin_ws# rospack find catkin
terminate called after throwing an instance of 'rospack::Exception'
  what():  error parsing manifest of package bondcpp at /opt/ros/lunar/share/bondcpp/package.xml
Aborted
```